training/horovod/base/Dockerfile

```diff
@@ -1,18 +1,18 @@
 FROM gcr.io/deeplearning-platform-release/tf-gpu.1-15
 
 # Install OpenSSH, CUDA 10.0, NCCL and OpenMPI 4.0
+# Install Horovod, temporarily using CUDA stubs
 RUN apt-get update && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-change-held-packages \
-        openssh-client openssh-server cuda-toolkit-10-0 libnccl2 libnccl-dev && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends --allow-change-held-packages \
+        iperf openssh-client openssh-server cuda-toolkit-10-0 libnccl2 libnccl-dev && \
     wget https://download.open-mpi.org/release/open-mpi/v4.0/openmpi-4.0.2.tar.gz && \
     tar -xf openmpi-4.0.2.tar.gz && \
     cd openmpi-4.0.2 && \
     ./configure --enable-static --prefix=/usr --enable-shared --with-cuda=/usr/local/cuda/ && \
     make -j && make install && \
-    cd .. && rm -rf openmpi-4.0.2
-
-# Install Horovod, temporarily using CUDA stubs
-RUN ldconfig /usr/local/cuda/targets/x86_64-linux/lib && \
+    cd .. && rm -rf openmpi-4.0.2 && \
```

Comment:
Do we still need this with tfe base image?

Reply:
We still need this because the base image does not have openmpi installation and the version in apt is openmpi 2.0 which is too old.

Reply:
I thought horovod is preinstalled in the image. If that doesn't work we can follow up with a fix on the image level.

Reply:
There is no horovod installation in the image. Agreed that it should be included.